src/main/java/spark/RequestResponseFactory.java

```diff
@@ -26,6 +26,13 @@
     private RequestResponseFactory() {
     }
 
+    /**
+     * Solely used for custom error pages.
```

Comment:
I'm not sure this comment is helpful. Could the method be used for anything else? If not, call it something that contains the words "custom error pages", and remove the comment?